src/response/cache_sorted_set_fetch_response.rs

```diff
@@ -1,7 +1,159 @@
-use momento_protos::cache_client::SortedSetElement;
+use crate::sorted_set::Elements;
+use crate::MomentoError;
+use core::convert::TryFrom;
+use momento_protos::cache_client::sorted_set_fetch_response::SortedSet;
 
 #[derive(Debug)]
 #[non_exhaustive]
 pub struct MomentoSortedSetFetchResponse {
-    pub value: Option<Vec<SortedSetElement>>,
+    pub value: Option<SortedSet>,
+}
```

Comment:
apologies for any confusion / learning curve on my part as I am coming up to speed on this repo. But looking at Kenny's implementation for DictionaryFetch I think you're right, this struct goes away in favor of the enum below?

Reply:
Correct - I should revert this. We'll leave this type in-place until the old `sorted_set_fetch` function is removed

Reply:
aha, I see. Makes sense.